k8s/examples/kafka/89-kafka-test-client.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kafka-test-client
+  namespace: default
```

Comment:
Could we please remove the namespace: default.

Reply:
removed